lib/addons/cloudwatch-adot-addon/index.ts

```diff
@@ -73,6 +75,8 @@ export class CloudWatchAdotAddOn implements ClusterAddOn {
     }
 
     @dependable(AdotCollectorAddOn.name)
+    @dependable(KubeStateMetricsAddOn.name)
+    @dependable(PrometheusNodeExporterAddOn.name)
```

Comment:
Is it really a dependency? Do you not envision a case when customers may want to use the CW adot addon without it?

If it is an "optional" dependency we have a different path of handling it.

Reply:
Its not required for CW Addon. Removed it.